sdk/apis/kubebind/v1alpha2/apiresourceschema_types.go

```diff
@@ -0,0 +1,247 @@
+/*
+Copyright 2025 The Kube Bind Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha2
+
+import (
+	"encoding/json"
+
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+// InformerScope is the scope of the Api.
+//
+// +kubebuilder:validation:Enum=Cluster;Namespaced
+type InformerScope string
+
+const (
+	ClusterScope    InformerScope = "Cluster"
+	NamespacedScope InformerScope = "Namespaced"
+)
+
+// APIResourceSchema
+// +crd
+// +genclient
+// +genclient:nonNamespaced
```

Comment:
This refers to my and @sttts  conv here 
https://docs.google.com/document/d/1pWFaJbEyBJVERkpm-saFMsGs9WEPYEIX2jFtusR5-do/edit?disco=AAABdQvmdLk

I still want to see this Namespaced for possible soft separation if needed. 
Like `v1` in namespace `foo`, and `v2` in namespace `bar` type of thing. 

but in my head it just not a blocker either way 🤷

Reply:
For kube-bind I can see it more suitable to have it namespaced so that I would agree on this